### PR TITLE
[Translation: Japanese] [Stype] Reducing number of chars for #263.

### PR DESCRIPTION
**Pull request by nicky008**
From #263, each ship type word (kanji) in Japanese will has only 3 chars in maximum number now.

### CODE REVIEW

File: src/data/translations/jp/stype.json [hunk 1]
```diff
@@ -2,23 +2,23 @@
...
-    "軽巡洋艦",
-    "重雷装巡洋艦",
-    "重巡洋艦",
-    "航空巡洋艦",
+    "軽巡洋",
+    "重雷装",
+    "重巡洋",
+    "航空巡",
     "軽空母",
     "F戦艦",
```
Comment (bamboo3250) [inline src/data/translations/jp/stype.json]: Do the japanese use any kanji for "Fast" in Fast Battleship or just F?

**Comment by Crazyteitoku** [comment]
@bamboo3250 I know in the game technically it is meant to be 高速戦艦 (kou soku sen kan) which literally means fast speed battleship when it comes to describing fast battleships. Not sure elsewhere however.

**Comment by nicky008** [comment]
Frankly, their is this word "高速戦艦", directly mean "fast battleship". But in Kancolle, the devs still retained to use "戦艦" for any fast battleships here for now.

I personally want to use "高戦艦" for fast battleship in my "3 chars rule" if most of everyone agreed.

**Comment by bamboo3250** [comment]
If that is the case then we should stick with F戦艦. I cannot find any document that uses the word 高戦艦 or 速戦艦 for fast battleships. Perhaps we cannot separate the word 高速.

**Comment by LostLogia4** [comment]
@bamboo3250 is right. "高" means tall or high and has nothing to do with speed. "速" means speed, but it has no sense of magnitude on its own, so individually neither kanji could represent "high speed" by itself.

**Comment by dragonjet** [comment]
I guess I'm the only one with zero japanese :3

**Comment by nicky008** [comment]
@LostLogia4 @bamboo3250 Thx for your suggestions...

**Comment by bamboo3250** [comment]
No need for that. You did it correctly from the start though :stuck_out_tongue_closed_eyes:

**Comment by c933103** [comment]
wouldn't 
 "軽巡",
   "重雷巡",
  "重巡",
    "航巡",
work better?

**Comment by nicky008** [comment]
@c933103 Nice suggestion, I had (at least once) thought these words should be in this (巡) from as suffix.

**Pull request merged**